CHANGES.rst

```diff
@@ -5,6 +5,8 @@ CHANGES
 
 Unreleased
 ----------
+- CSS: Adjusted inner alignment being off for cards with links
```

Comment:
what do you mean `cards` here? (Apologies, not familiar with the terminology)

Reply:
Hi. Apologies. A "card" is a visual design element. Google's Material Design heavily uses it, and also made them more popular. However, it's an ubiquitous element these days, not exclusively tied to Material any longer.

-- https://sphinx-design.readthedocs.io/en/latest/cards.html
-- https://m3.material.io/components/cards/overview

Reply:
thx!